pkg/server/router/presentation.go

```diff
@@ -40,13 +42,6 @@ type CreatePresentationDefinitionRequest struct {
 	Format                 *exchange.ClaimFormat            `json:"format,omitempty" validate:"omitempty,dive"`
 	InputDescriptors       []exchange.InputDescriptor       `json:"inputDescriptors" validate:"required,dive"`
 	SubmissionRequirements []exchange.SubmissionRequirement `json:"submissionRequirements,omitempty" validate:"omitempty,dive"`
```

Comment:
I think having author is still valuable without kid

Reply:
Author is not part of a Presentation Definition. Adding things that aren't part of the spec makes it confusing for users.

Additionally, author moved to be the `IssuerID` field in the Presentation Request.